collect_app/src/androidTest/java/org/odk/collect/android/instrumented/utilities/ImageConverterTest.java

```diff
@@ -419,7 +338,7 @@ public void scaleImageToNewWidthTest() {
     }
 
     @Test
-    public void keepExifTest1AfterScaleAndRotation() {
+    public void keepExifTest1AfterScaling() {
```

Comment:
A better name would be just `keepExifAfterScaling`

Reply:
Ah ok, I just reused the naming convention how it was. Will fix this